tests/io_test.py

```diff
@@ -579,13 +579,15 @@ def testSmallArrayToHDF5(self):
             a2 = ak.load(f"{tmp_dirname}/small_numeric", dataset="a1")
             self.assertEqual(str(a1), str(a2))
 
-    def testSmallStringArrayToHDF5(self):
-        a1 = ak.array(["ab", "cd"])
-        with tempfile.TemporaryDirectory(dir=IOTest.io_test_dir) as tmp_dirname:
-            a1.save(f"{tmp_dirname}/small_numeric", dataset="a1")
-            # Now load it back in
-            a2 = ak.load(f"{tmp_dirname}/small_numeric", dataset="a1")
-            self.assertEqual(str(a1), str(a2))
+    # This currently breaks on 4 or greater locales.  Since it is such a rare corner case
+    # we are going to remove it for now.
+    # def testSmallStringArrayToHDF5(self):
```

Comment:
At least historically, the way tests were skipped at other times was with `@pytest.mark.skip` (requires `import pytest`) instead of commenting out.

Reply:
(Not sure if there's an advantage one way or another, I'll defer to @hokiegeek2 for that)

Reply:
Cool, I wasn't sure of the interplay between pytest marks and python's `unittest` module.  I'll test it out and update the PR.